Jenkinsfile

```diff
@@ -14,66 +14,69 @@ elifePipeline {
+        //                 try {
+        //                     sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml run --name elife-xpub_app_style_guide app npm run build:styleguide"
+        //                     sh "docker cp elife-xpub_app_style_guide:/home/xpub/_build_styleguide ${folder}"
+        //                     sh "aws s3 cp --recursive ${folder} s3://ci-elife-xpub-styleguide/${folder}"
+        //                     echo "Styleguide URL: $targetUrl"
+        //                 } catch (e) {
+        //                     sh "docker-compose -f docker-compose.yml -f docker-compose.ci.yml down -v"
+        //                 }
+        //             },
+        //             [
+        //                 'name': 'styleguide',
+        //                 'commit': commit,
+        //                 'targetUrl': targetUrl,
+        //             ]
+        //         )
+        //     }
 
-            try {
-                sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml up -d postgres"
-                sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml run --rm --name elife-xpub_wait_postgres app bash -c './scripts/wait-for-it.sh postgres:5432'"
-                parallel actions
-            } finally {
-                sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml down -v"
-            }
-        }
+        //     try {
+        //         sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml up -d postgres"
+        //         sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml run --rm --name elife-xpub_wait_postgres app bash -c './scripts/wait-for-it.sh postgres:5432'"
+        //         parallel actions
+        //     } finally {
+        //         sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml down -v"
+        //     }
+        // }
 
         stage 'Browser Tests', {
             try {
                 sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml up -d postgres"
                 sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml run --rm --name elife-xpub_wait_postgres app bash -c './scripts/wait-for-it.sh postgres:5432'"
+                sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml run --rm --name elife-xpub_clobber_db app bash -c 'npx pubsweet setupdb --clobber --username test --password password --email test@example.com'"
+                sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml up -d app"
+                sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml run --rm --name elife-xpub_wait_postgres app bash -c './scripts/wait-for-it.sh -t 180 localhost:3000'"
```

Comment:
this container is named `elife-xpub_wait_postgres` but it waits on the `app` (port 3000) instead

Reply:
fixed!